_includes/selector.html

```diff
@@ -457,6 +457,7 @@
             },
             getAdditionalPkgName(pkg) {
                 var pkg_names = {
+                    "NetworkX + nx-cugraph": "networkx nx-cugraph",
```

Comment:
We need a version pinning on nx-cugraph. All the RAPIDS packages get `=24.08` or `=24.10`. Copy that here if nx-cugraph is versioned the same way.
```suggestion
                    "NetworkX + nx-cugraph": "networkx nx-cugraph",
```

Reply:
I confirmed that the RAPIDS metapackage includes nx-cugraph, so do we still need to add `nx-cugraph` to the install line if it also includes `rapids`?

Reply:
Technically we do not. We only need it if we're selecting individual packages, as the `rapids` metapackage would transitively include `nx-cugraph`. However, it'd be an unexpected UX for a button labeled "NetworkX + nx-cugraph" to not add `nx-cugraph` explicitly. I would leave the behavior as-is.

Reply:
Version pinning changes for both Stable and Nightly:

![image](https://github.com/user-attachments/assets/0eebcbd8-18ed-4a27-81f9-a054e608f42f)

![image](https://github.com/user-attachments/assets/fa8e86cb-8ec5-4d5f-8cbc-6442d7336950)